### PR TITLE
Use Tycho for building Web 2

### PR DESCRIPTION
**Pull request by ctron**
This set of changes converts the build of the Web2 bundle to an actual tycho build. This allows to build all Kura bundles in one build reactor thus allowing us to build a final P2 repository in the end.

### CODE REVIEW

**Comment by MMaiero** [comment]
I had a look, but I have few questions. Could we discuss about this tomorrow?

**Comment by ctron** [comment]
Sure we can do that. If you already have some feedback, just let me know.

**Comment by ctron** [comment]
So, as just discussed, the bundle is now named `.web2`

File: kura/distrib/pom.xml [hunk 1]
```diff
@@ -350,13 +350,13 @@
 						</configuration>
 					</execution>
 					<execution>
-						<id>get-web2-bundle</id>
+						<id>get-web-bundle</id>
```
Comment (MMaiero) [inline kura/distrib/pom.xml]: Just to keep consistency with what we agreed: why not keeping it as web2?

File: kura/distrib/pom.xml [hunk 2]
```diff
@@ -350,13 +350,13 @@
...
 						<phase>generate-resources</phase>
 						<goals>
 							<goal>copy</goal>
 						</goals>
 						<configuration>
-							<skip>${org.eclipse.kura.web2.skip}</skip>
+							<skip>${org.eclipse.kura.web.skip}</skip>
```
Comment (MMaiero) [inline kura/distrib/pom.xml]: Same as before

File: kura/distrib/src/main/ant/build_equinox_distrib.xml [hunk 3]
```diff
@@ -121,8 +121,8 @@
 		<antcall target="wires-config" />
 
 		<!-- Conditional entries. Check for jar presence, and invoke relevant ant target-->
-		<available file="${project.build.directory}/plugins/org.eclipse.kura.web2_${org.eclipse.kura.web2.version}.jar" property="web2.jar.present" />
-		<antcall target="web2-config" />
+		<available file="${project.build.directory}/plugins/org.eclipse.kura.web2_${org.eclipse.kura.web2.version}.jar" property="web.jar.present" />
```
Comment (MMaiero) [inline kura/distrib/src/main/ant/build_equinox_distrib.xml]: Same as in kura/distrib/pom.xml

**Comment by ctron** [comment]
I thought we were only talking about the bundle itself.

Better that way?

**Pull request merged**